### DISCUSSION TITLE
Universal Package format

### DISCUSSION THREAD

**Discussion by JJ4884** [crosswire/xiphos#1081] [Q&A]
IS there any interest in bringing the Xiphos app to a universal package format such as Flatpak, AppImage, and/or Snap packages to make it easier for installation on a larger number of distros?

**Comment by alerque** [comment]
This question is already asked an answered, as can bee seen by [searching issues](https://github.com/crosswire/xiphos/issues?q=is%3Aissue+flatpak+is%3Aclosed). A Flatpak version was already created, but has been nothing but trouble to maintain and use. See #823 for where it was introduced and other issues for the troubles that came up.

With most major distro packaging worked out right now using system libraries, it is unclear what advantage would be gained by adding any (not so universal) container packaging format. What distro are you using that doesn't have packages?

**Reply by domcorbex** [reply]
`What distro are you using that doesn't have packages?`

Maybe Fedora Silverblue? 
(Fedora Silverblue is an immutable operating system, applications are installed in their flatpak -containerized version)

**Reply by alerque** [reply]
Wow where did that come from? I have *not* been paying attention to the Fedora/RedHat/CentOS scene lately.

**Reply by greg-hellings** [reply]
Been around for several years. It grew out of several different movements, but all inspired by CoreOS and the Atomic Server response from Red Hat and Fedora.

I use it as my daily driver.
